### PR TITLE
Create 2020-02-11-night-shift-f-lux.md

### PR DESCRIPTION
**Pull request by valeIT**

### CODE REVIEW

File: _posts/2020-02-11-night-shift-f-lux.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+status: publish
+categories: [Tech]
+image:
+image2:
+author: Valentino Urbano
+---
+
+*I started answering to @brandonwilson 's comment, but felt that it was getting out of hand as a simple comment.*
+
+Night Shift from Apple has all the basics covered out: it shifts the hue of your screen to make it easier to the eyes during night time and has less adverse effects on your sleep cycle. I'm not sure it actually works in getting a better sleep at night, but it is surely beneficial on not getting watery and itchy eyes if you use your pc/mac a lot in the evenings. I used to do that, but during the last 6+ months I've been stopping at 10 pm max. I'm getting way longer sleep thanks to that (but that's a story for another time).
```
Comment (MsSlacker) [inline _posts/2020-02-11-night-shift-f-lux.md]: to the eyes - > for?

File: _posts/2020-02-11-night-shift-f-lux.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+I started using it way before it became a standard OS component thanks to f.lux which has been around since 2009.
+
+Nowadays I'm still using f.lux instead of the default one since it provides way more customizabily while being free. The killer feature is being able to customize the hue to whatever you prefer from several presets.
```
Comment (MsSlacker) [inline _posts/2020-02-11-night-shift-f-lux.md]: customizabily - > customizability

Comment (MsSlacker) [reply _posts/2020-02-11-night-shift-f-lux.md]: while still being free?

File: _posts/2020-02-11-night-shift-f-lux.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+[From f.lux developers:][1]
+>Every person has individual needs, and those needs are different based  on your sensitivity to light, your own chronobiology (imagine early  birds and night owls), your own schedule, and other factors too. Those  needs change across seasons, and over your lifetime. Today our approach  is different: we are working every day to understand how light affects  human biology, not strictly sleep, and we are constantly applying what  we learn to updates and new features for f.lux.
+
+If you want to go more in depth on the differences I'd suggest [to read this article from iMore][2].
+
+Unfortunately on iOS you are forced to use Night Shift since due to Apple limitation an application like f.lux can't function on the platform. In the past when the iOS was very limited and jailbreaking was actually useful I have used f.lux from [Cydia][3], but nowadays there is not much point in jailbreaking and doing it just for f.lux seems wasteful. Night Shift is good enough and that's all that Apple cares about at the moment.
```
Comment (MsSlacker) [inline _posts/2020-02-11-night-shift-f-lux.md]: limitation - > s?

**Review by MsSlacker — CHANGES_REQUESTED** [review]

File: _posts/2020-02-11-night-shift-f-lux.md (hunk 3 above)
Comment (valeIT) [reply _posts/2020-02-11-night-shift-f-lux.md]: commas as well

**Review by MsSlacker — APPROVED** [review]

**Pull request merged**